xbmc/utils/StringUtils.cpp

```diff
@@ -1827,6 +1828,20 @@ uint64_t StringUtils::ToUint64(const std::string& str, uint64_t fallback) noexce
   return result;
 }
 
+int StringUtils::ToInt(const std::string_view str, int fallback /* = 0 */) noexcept
```

Comment:
as far as I understand this will return always an int even if the conversion fails. Do you want to add the overlay on those cases? I'd say if a conversion fails you should abort the processing of the said overlay/subtitle

Reply:
using from_chars gives greater security but does not improve anything,
all values are processed with regex which already performs all the checks of the data type therefore the parsing will never fail,
in case the webvtt file does not follow the standard specs these situations are already handled

Reply:
add two new methods

Reply:
from_chars isnt full implemented on all compilers then double fails
need to find another way

Reply:
just keep it as it was originally if it was safe already